system/iptables.py

```diff
@@ -220,6 +224,11 @@ def append_param(rule, param, flag, is_list):
         if param is not None:
             rule.extend([flag, param])
 
+def append_comm(rule, param):
```

Comment:
I think there should be one extra blank line above this, please run your code thru PEP8. [online PEP8 checker](http://pep8online.com)